### ISSUE TITLE
Wildcards character in publicPath not working for uploading source map

### ISSUE THREAD

**Issue by b-dur** [bugsnag/webpack-bugsnag-plugins#35]
The _resolve_ function from _url_ package doesn't support wildcard characters.

**Problem explaining**
Let's assume we have a project, that outputs _scripts.js_ and _scripts.js.map_ in the root dist folder. The code is served from a subdomain of example.com, e.g. app.example.com and the files would be available at _https://app.example.com/scripts.js_ and _https://app.example.com/scripts.js.map_

**Webpack config**
``` js
  new BugsnagSourceMapUploaderPlugin({
    apiKey: 'YOUR_API_KEY',
    publicPath: 'https://*.example.com/',
  })
```

**Expected**
Source map files would be uploaded where url matches: `https://*.example.com/scripts.js`
**Actual**
The url that is being matched is `https:///*.example.com/scripts.js` (note the extra `/` before the wildcard).

**Problem shooting**
Turns out that the `resolve` function of the `url` package doesn't support `*` and hence resolves the URL to something unexpected. See source-map-uploader-plugin.js [line 77](https://github.com/bugsnag/webpack-bugsnag-plugins/blob/master/source-map-uploader-plugin.js#L77)

Changing publicPath in the webpack config produces following results:
`publicPath: 'https:/*.example.com/'` results `https:/*.example.com/scripts.js`
`publicPath: 'https://*.example.com'` results `https:///scripts.js`

Guess the `publicPath: 'https:/*.example.com/'` would work, since the wildcard in the result would then include the missing slash `/` in the protocol. However that would be a hack to circumvent the problem.

**Comment by mattdyoung** [comment]
Hi @b-dur 

Thanks for the report. We just recently became aware of this issue ourselves and are just scheduling a fix which we should be able to implement and release in the near future.

Thanks
Matt

**Comment by abigail-bugsnag** [comment]
@b-dur this has now been fixed in v1.4.1
